tests/test_classes.py

```diff
@@ -12,20 +12,33 @@
 from iceqube.utils import stringify_func
 from iceqube.storage import Storage
 
+from sqlalchemy import create_engine
+from sqlalchemy.pool import NullPool
+
 
 @pytest.fixture
 def backend():
     with tempfile.NamedTemporaryFile() as f:
-        b = Storage(app="pytest", namespace="test", storage_path=f.name)
+        connection = create_engine(
+            "sqlite:///{path}".format(path=f.name),
+            connect_args={'check_same_thread': False},
+            poolclass=NullPool,
+        )
+        b = Storage(app="pytest", namespace="test", connection=connection)
         yield b
         b.clear()
 
 
 @pytest.fixture
 def inmem_queue():
     with tempfile.NamedTemporaryFile() as f:
-        e = Worker(app="pytest", storage_path=f.name)
-        c = Queue(app="pytest", storage_path=f.name)
+        connection = create_engine(
+            "sqlite:///{path}".format(path=f.name),
+            connect_args={'check_same_thread': False},
+            poolclass=NullPool,
+        )
+        e = Worker(app="pytest", connection=connection)
```

Comment:
`app` is defined as a positional argument in the constructor...

```suggestion
        e = Worker("pytest", connection=connection)
```